spec/spec_helper.rb

```diff
@@ -10,8 +10,8 @@
     puppetversion: Puppet.version,
     facterversion: Facter.version,
     ipaddress: '172.16.254.254',
-    macaddress: 'AA:AA:AA:AA:AA:AA'
+    macaddress: 'AA:AA:AA:AA:AA:AA',
   }
 end
-require 'spec_helper_methods'
+ require 'spec_helper_methods'
```

Comment:
I didn't edit that file manually so the best I can figure is that it's an artifact of something. I can't put my finger on what, though.